mdbook-tera-backend/src/tera_renderer/error.rs

```diff
@@ -0,0 +1,25 @@
+use std::path::StripPrefixError;
+
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum RendererError {
+    #[error("IO Error: {0}")]
+    IoError(#[from] std::io::Error),
+    #[error("Invalid path: {0}")]
+    InvalidPath(String),
+    #[error("Error rendering tera template: {0}")]
+    TeraError(#[from] tera::Error),
+    #[error("HTML Error: {0}")]
+    HtmlRewritingError(#[from] lol_html::errors::RewritingError),
+    #[error("Mdbook Error: {0}")]
+    Mdbook(#[from] mdbook::errors::Error),
+    #[error("Error in strip_prefix call: {0}")]
+    StripPrefixError(#[from] StripPrefixError),
+    #[error("Serde error: {0}")]
+    SerdeError(#[from] serde_json::Error),
+    #[error("Serde error: {0}")]
+    DependencyNotFound(String),
+}
```

Comment:
Similar to above, we should not have to maintain this kind of code. Every line of code is a liability, so we should avoid writing more than we have to.

Reply:
We could use anyhow, but I'm not a fan

Reply:
We should just remove it, the code here should not be so complex that we need this much infrastructure.

Reply:
Sounds good, let's just use anyhow